src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/callbacks/caseworker/MoreTimeRequestedCallbackHandler.java

```diff
@@ -72,13 +72,13 @@
     private static final String PREVIEW_SENTENCE = "The response deadline will be %s .";
     private static final String ERROR_MESSAGE =
         "There was a technical problem. Nothing has been sent. You need to try again.";
-    private static final String STANDARD_DEADLINE_TEXT = "You’ve been given an extra 14 days to respond to the"
-            + " claim made against you by %s.\n"
-            + "\n"
-            + "You now need to respond to the claim before 4pm on %s.\n"
-            + "\n"
-            + "If you don’t respond, you could get a County Court Judgment (CCJ). This may make it harder to get "
-            + "credit, such as a mobile phone contract, credit card or mortgage.";
+    private static final String STANDARD_DEADLINE_TEXT = String.join(newLine,
```

Comment:
This wont compile as there is no `newLine` variable. However you could take @andrewwa-kainos's advice and use `%n` i.e 
```suggestion
    private static final String STANDARD_DEADLINE_TEXT = String.join("%n",
```